src/rules.ts

```diff
@@ -133,7 +133,7 @@ export const rules: Rule[] = [
         new Example(
             'Removes trailing spaces and tabs',
             dedent`
-        # H1   
```

Comment:
revert the trailing spaces in this example

Reply:
Fixed